### ISSUE TITLE
aria-live test results

### ISSUE THREAD

**Issue by ZoeBijl** [accessibilitysupported/a11ysupport.io#101]
Today I tested `aria-live` in a variety of ATs, browsers, and operating systems. If you want you can use [my test result data](https://moiety.me/research/aria-live/) for this project.

**Comment by mfairchild365** [comment]
@ZoeBijl this is great! Thank you so much! `aria-live` is near the top of my priority list to test & document, so this is very timely and valuable.

**Comment by mfairchild365** [comment]
See related #85

**Comment by mfairchild365** [comment]
We now have our own suite of aria-live tests. If you feel that something is missing, please let us know. Thank you again for offering your data @ZoeBijl :) We ended up creating our own tests to better match the structure of this project.
